jobs/rep/templates/bpm.yml.erb

```diff
@@ -9,6 +9,8 @@ processes:
   ephemeral_disk: true
   additional_volumes:
     - path: <%= p("diego.executor.volman.driver_paths") %>
+    - path: <%= p("diego.executor.volman.csi_paths") %>
```

Comment:
Since this is a `:` or `;` delimited list (as per [this property description](https://github.com/cloudfoundry/diego-release/blob/52abb133ab04b028a55b912b8e6b1be122de18ad/jobs/rep/spec#L195-L197)), should this `erb` interpolation be expanded to split this path and add a separate list element for each member of the list?

Reply:
@sunjayBhatia I was talking to @emalm about that last week.  In practice, even though we call this out as a list, we only ever pass a single value.  So our preference is to leave it as-is, and then later when the rep properties get flattened out, we can introduce the new flattened property as a single path.  (This applies to both `driver_paths` and `csi_paths`.)